Engine/source/console/console.cpp

```diff
@@ -307,6 +307,9 @@ bool alwaysUseDebugOutput = true;
 bool useTimestamp = false;
 bool useRealTimestamp = false;
 
+static U32 initTime = Platform::getRealMilliseconds();
+U32 startTime = initTime;
```

Comment:
should be `static U32 startTime = initTime;` to avoid name collision outside of source file.